src/prototype_room_creepbuilder.js

```diff
@@ -195,9 +195,11 @@ Room.prototype.checkRoleToSpawn = function(role, amount, targetId, targetRoom, l
   if (this.inQueue(creepMemory) || this.inRoom(creepMemory, amount)) {
     return false;
   }
-  if (targetRoom !== base && typeof Game.map.getRoomStatus(targetRoom) !== 'undefined' && typeof Game.map.getRoomStatus(base) !== 'undefined') {
-    if (Game.map.getRoomStatus(targetRoom).status !== Game.map.getRoomStatus(base).status) {
-      return false;
+  if (targetRoom !== base) { // should fix ci Error: No runtime status data
+    if (Game.time > 230 && typeof Game.map.getRoomStatus(targetRoom) !== 'undefined' && typeof Game.map.getRoomStatus(base) !== 'undefined') {
```

Comment:
I added a try catch block.

But `Game.map.getRoomStatus` should only be called once and stored as variable, to save CPU and looks cleaner.